### PR TITLE
Add option to show the GUI on a specific monitor

### PR DESCRIPTION
**Pull request by Cobinja**
This adds the option "only-on-monitor".
Default is "auto", which means "Follow the mouse", like it was without this option.
PR for slick-greeter is https://github.com/linuxmint/slick-greeter/pull/76

### CODE REVIEW

**Comment by kneekoo** [comment]
Hello, any news on this one?

**Comment by Cobinja** [comment]
Rebased to current master

**Comment by clefebvre** [comment]
Hi @Cobinja,

Please don't rely on cinnamon/mate/gnome desktop libs for this. I know they add the ability to show the monitor names (as opposed to just the plug names), but that's an unfortunate (even if loose) dependency here and it makes testing more complicated.

I'd much prefer something that works the same way everywhere. You can get the plug names via Gdk, that's how you get them in slick itself:

```
screen = Gdk.Screen.get_default()
display = screen.get_display()
primary = display.get_primary_monitor()
for i in range(0, display.get_n_monitors()):
	monitor = display.get_monitor(i)
	print(monitor.get_model())

```

In your msgids, beware of "shown", it sounds weird in the US, just like "showed" sounds weird here. Can you rephrase this and avoid that word altogether?
